test/go.mod

```diff
@@ -2,9 +2,9 @@ module sigs.k8s.io/cluster-api-provider-vsphere/test
 
 go 1.22.0
```

Comment:
We should also update the top-level go.mod

Reply:
Missed, let me fix it.

Reply:
Ah nice, but test is green 🎉